doc/devdocs/modules/launcher/new-plugin-checklist.md

```diff
@@ -0,0 +1,30 @@
+# New plugin checklist
+- A plugin is a project under `modules\launcher\Plugins`
+- Microsoft plugin project name pattern: `Microsoft.PowerToys.Run.Plugin.{PluginName}`
+- Community plugin project name pattern: `Community.PowerToys.Run.Plugin.{PluginName}`
+- [`GlobalSuppressions.cs`](/src/codeAnalysis/GlobalSuppressions.cs) and [`StyleCop.json`](/src/codeAnalysis/StyleCop.json) have to be included in the plugin project so it follows PowerToys code guidelines
+- A plugin has to have `{PowerToys version}.0` version
+- A plugin has to have only x64 platform target
```

Comment:
wouldn't this break us into jail for when Arm64 support comes?

Reply:
We support only x64. So we should not specify other targets in a `*.csproj` file. We will add those targets when porting to other platform targets.

Reply:
Rephrased message to make it less confusing.

Reply:
To make it 100% clear: this is not cause any problem with Arm64. And this is what we have been doing for all projects.